ert_logging/logger.conf

```diff
@@ -0,0 +1,28 @@
+version: 1
```

Comment:
This logger is disjoint from all other loggers in `libres`?

Reply:
The only thing worth mentioning in libres is the ert_logger module, which is the one posting to devnull. As for anything other logger in libres, they are coming from the C layer, which is not something we wish to deal with. This is just configuring the default python logging module. If the config is loaded prior to using logging module, you will get that config. There is no other proper configuration of the logging module in either libres nor ert.